client/src/components/Payments/PaymentsPage.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { loadStripe } from '@stripe/stripe-js';
+import { Elements } from '@stripe/react-stripe-js';
+
+import CardForm from './CardForm';
+
+const PaymentsPage = (): JSX.Element => {
+  const makeStripePromise = async () => {
+    const response = await fetch('/payments/public-key');
```

Comment:
The publishable key can be stored in an env on the FE :)

Reply:
That was easier :)) Didn't think I could also do that for front end. Thanks!